community/values/src/main/java/org/neo4j/values/storable/Values.java

```diff
@@ -253,6 +257,40 @@ public static ShortArray shortArray( short[] value )
         return new ShortArray.Direct( value );
     }
 
+    public static PointValue pointValue( CoordinateReferenceSystem crs, double... coordinate )
+    {
+        return new PointValue( crs, coordinate );
+    }
+
+    public static PointValue point( Point point )
+    {
+        // An optimization could be to do an instanceof PointValue check here
+        // and in that case just return the casted argument.
+        List<Double> coordinate = point.getCoordinate().getCoordinate();
```

Comment:
".getCoordinate().getCoordinate()" looks quite strange

Reply:
It does indeed. And sadly this is part of the public interface since 3.0. Personally I think that interface was not perfectly designed and would like to change it. In more ways than this one. Current policy is to only make changes on major releases. I wonder if we could, for a possibly entirely unused interface like this, motivate to make changes on minor releases? Worth discussing.